versioned_docs/version-22.04/graph-views/map-web-update.md

```diff
@@ -14,30 +14,58 @@ Use the following procedure to update your MAP version:
   ```
 
 2. Update the packages by running this command on the machine(s) hosting the central service and the Centreon MAP service:
+
+<Tabs groupId="sync">
+<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
+
+``` shell
+sudo dnf update centreon-map-engine centreon-map-web-client
+```
+
+</TabItem>
+<TabItem value="CentOS 7" label="CentOS 7">
+
+``` shell
+sudo yum update centreon-map-engine centreon-map-web-client
+```
+
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+ - If MAP only is installed:
+ 
+ ``` shell
+ sudo apt upgrade centreon-map-engine centreon-map-web-client
+ ```
   
-  <Tabs groupId="sync">
-  <TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
-  
-  ``` shell
-  sudo dnf update centreon-map-engine centreon-map-web-client
-  ```
-  
-  </TabItem>
-  <TabItem value="CentOS 7" label="CentOS 7">
-  
-  ``` shell
-  sudo yum update centreon-map-engine centreon-map-web-client
-  ```
-  
-  </TabItem>
-  <TabItem value="Debian 11" label="Debian 11">
-  
-  ``` shell
-  sudo apt update centreon-map-engine centreon-map-web-client
-  ```
-  
-  </TabItem>
-  </Tabs>
+ - If MAP and MAP Legacy are installed on the same server:
+   
+   - Make a backup of the **map.cnf** file:
+    
+    ```shell
+    cp /etc/mysql/map.cnf /etc/mysql/map.cnf.bk
```

Comment:
The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf /etc/my.cnf.d/map.cnf.bk`